aws/data_source_aws_ebs_volume_ids_test.go

```diff
@@ -0,0 +1,105 @@
+package aws
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccDataSourceAwsEbsVolumeIDs(t *testing.T) {
+	rInt := acctest.RandIntRange(0, 256)
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckEbsVolumeIDsDestroy,
```

Comment:
We can just reuse the resource `CheckDestroy` here instead of creating a new function. 👍 

```suggestion
		CheckDestroy: testAccCheckVolumeDestroy,
```